### PR TITLE
CA-1512: tags subtype proposal

### PR DESCRIPTION
**Pull request by wapo-omar-khan**

### CODE REVIEW

File: docs/proposals/2019-05-13_Tags_Subtype.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Example
+```json
+{
+  "_id": "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+  "type": "story",
+  "version": "0.10.1",
+
+  "taxonomy": {
+    "tags": {
```
Comment (anscho) [inline docs/proposals/2019-05-13_Tags_Subtype.md]: Should this be `tag` or an array?

**Comment by jpblair** [comment]
Does this change anything other than adding a `subtype` field to the existing tag objects?

**Comment by gengel** [comment]
@jpblair No

File: docs/proposals/2019-05-13_Tags_Subtype.md (hunk 1 above)
Comment (wapo-omar-khan) [reply docs/proposals/2019-05-13_Tags_Subtype.md]: according to the schema, it's an object called tags

Comment (wapo-omar-khan) [reply docs/proposals/2019-05-13_Tags_Subtype.md]: and i just remembered that elasticsearch mappings consider elements and arrays of that element the same type. arrays is right

**Pull request merged**